packages/Meshfree/test/tstDetailsCommunicationHelpers.cpp

```diff
@@ -123,44 +125,51 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( DetailsDistributedSearchTreeImpl,
-    Kokkos::parallel_for( Kokkos::RangePolicy<ExecutionSpace>( 0, comm_size ),
-                          KOKKOS_LAMBDA( int i ) {
-                              for ( int j = tn( i ); j < tn( i + 1 ); ++j )
-                                  ranks_v( j ) = i;
-                          } );
-    Kokkos::fence();
-
-    Kokkos::View<int **, DeviceType> v_ref( "v_ref", comm_size * comm_rank,
-                                            DIM );
-    Kokkos::parallel_for(
-        Kokkos::RangePolicy<ExecutionSpace>( 0, comm_size * comm_rank ),
-        KOKKOS_LAMBDA( int i ) {
-            for ( int j = 0; j < DIM; ++j )
-                v_ref( i, j ) = j * comm_rank;
-        } );
-    Kokkos::fence();
-
-    Helper<DeviceType>::checkSendAcrossNetwork( comm, ranks_v, v_exp, v_ref,
-                                                success, out );
+    //    // send k packets to rank k
+    //    // receive k*comm_size packets
+    //    Kokkos::View<int *, DeviceType> tn( "tn", comm_size + 1 );
+    //    Kokkos::parallel_for(
+    //        Kokkos::RangePolicy<ExecutionSpace>( 0, comm_size + 1 ),
+    //        KOKKOS_LAMBDA( int i ) { tn( i ) = ( i * ( i - 1 ) ) / 2; } );
+    //    Kokkos::fence();
+    //
+    //    Kokkos::View<int **, DeviceType> v_exp( "v_exp", ArborX::lastElement(
+    //    tn ),
+    //                                            DIM );
+    //    Kokkos::parallel_for( Kokkos::RangePolicy<ExecutionSpace>( 0,
+    //    comm_size ),
+    //                          KOKKOS_LAMBDA( int i ) {
+    //                              for ( int j = tn( i ); j < tn( i + 1 ); ++j
+    //                              )
+    //                                  for ( int k = 0; k < DIM; ++k )
+    //                                      v_exp( j, k ) = i * k;
+    //                          } );
+    //    Kokkos::fence();
+    //
+    //    Kokkos::View<int *, DeviceType> ranks_v( "", ArborX::lastElement( tn )
+    //    ); Kokkos::parallel_for( Kokkos::RangePolicy<ExecutionSpace>( 0,
+    //    comm_size ),
+    //                          KOKKOS_LAMBDA( int i ) {
+    //                              for ( int j = tn( i ); j < tn( i + 1 ); ++j
+    //                              )
+    //                                  ranks_v( j ) = i;
+    //                          } );
+    //    Kokkos::fence();
+    //
+    //    Kokkos::View<int **, DeviceType> v_ref( "v_ref", comm_size *
+    //    comm_rank,
+    //                                            DIM );
+    //    Kokkos::parallel_for(
+    //        Kokkos::RangePolicy<ExecutionSpace>( 0, comm_size * comm_rank ),
+    //        KOKKOS_LAMBDA( int i ) {
+    //            for ( int j = 0; j < DIM; ++j )
+    //                v_ref( i, j ) = j * comm_rank;
+    //        } );
+    //    Kokkos::fence();
+    //
+    //    Helper<DeviceType>::checkSendAcrossNetwork( comm, ranks_v, v_exp,
+    //    v_ref,
+    //                                                success, out );
```

Comment:
Why did you comment all of this?

Reply:
That's when I was debugging my problem with `sendAcrossNetwork`. It should not be commented.